website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -0,0 +1,574 @@
+
+```Go
+// New returns a new Client using the provided config and Options.
+func New(config *rest.Config, options client.Options) (client.Client, error)
+```
+
+`client.Options` allow the caller to specify how the new Client should communicate with the API server.
+
+```Go
+// Options are creation options for a Client
+type Options struct {
+	// Scheme, if provided, will be used to map go structs to GroupVersionKinds
+	Scheme *runtime.Scheme
+
+	// Mapper, if provided, will be used to map GroupVersionKinds to Resources
+	Mapper meta.RESTMapper
+}
+```
+Example:
+```Go
+import (
+    "sigs.k8s.io/controller-runtime/pkg/client/config"
+    "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+cfg, err := config.GetConfig()
+...
+c, err := client.New(cfg, client.Options{})
+...
+```
+
+**Note**: defaults are set by `client.New` when Options are empty. The default [scheme][code-scheme-default] will have the [core][doc-k8s-core] Kubernetes resource types registered. The caller *must* set a scheme that has custom operator types registered for the new Client to recognize these types.
+
+Creating a new Client is not usually necessary nor advised, as the default Client is sufficient for most use cases.
+
+### Reconcile and the Client API
+
+A Reconciler implements the [`reconcile.Reconciler`][doc-reconcile-reconciler] interface, which exposes the Reconcile method. Reconcilers are added to a corresponding Controller for a Kind; Reconcile is called in response to cluster or external Events, with a `reconcile.Request` object argument, to read and write cluster state by the Controller, and returns a `reconcile.Result`. SDK Reconcilers have access to a Client in order to make Kubernetes API calls.
+
+**Note**: For those familiar with the SDK's old project semantics, [Handle][doc-osdk-handle] received resource events and reconciled state for multiple resource types, whereas Reconcile receives resource events and reconciles state for a single resource type.
+
+```Go
+// ReconcileKind reconciles a Kind object
+type ReconcileKind struct {
+	// client, initialized using mgr.Client() above, is a split client
+	// that reads objects from the cache and writes to the apiserver
+	client.Client
+
+    Log    logr.Logger
+
+	// scheme defines methods for serializing and deserializing API objects,
+	// a type registry for converting group, version, and kind information
+	// to and from Go schemas, and mappings between Go schemas of different
+	// versions. A scheme is the foundation for a versioned API and versioned
+	// configuration over time.
+    Scheme *runtime.Scheme
+}
+
+// Reconcile watches for Events and reconciles cluster state with desired
+// state defined in the method body.
+// The Controller will requeue the Request to be processed again if an error
+// is non-nil or Result.Requeue is true, otherwise upon completion it will
+// remove the work from the queue.
+func (r *ReconcileKind) Reconcile(request reconcile.Request) (reconcile.Result, error)
```

Comment:
Update this in the format of the new scaffold.
https://github.com/operator-framework/operator-sdk-samples/blob/master/go/kubebuilder/memcached-operator/controllers/memcached_controller.go#L46

Reply:
Update the comment above accordingly as well.